projects/rero/ng-core/src/lib/record/editor/object-type/object-type.component.ts

```diff
@@ -121,6 +121,9 @@ export class ObjectTypeComponent extends FieldType implements OnInit {
    * @returns boolean, true if the field can be hidden
    */
   canHide(field: FormlyFieldConfig) {
+    if (!this.field.templateOptions.longMode) {
+      return false;
+    }
```

Comment:
The modification of the other PR has been embedded into this commit, you can remove it.